frontend/doc-site/05__Annotate and Analyze Your Data/5_1__Getting Started: Install, Launch, Quick Start.mdx

```diff
@@ -130,7 +130,7 @@ pip install s3fs
 cellxgene launch s3://mybucket.s3-us-west-2.amazonaws.com/mydata.h5ad
 ```
 
-### Options for cellxgene launch
+### Options for CellxGene launch
```

Comment:
```suggestion
### Options for `cellxgene launch`
```

Note - code for CELLxGENE Annotate can/should remain `cellxgene` because that's a CLI invocation and is case sensitive.